src/main/webapp/js/client.js

```diff
@@ -155,3 +155,111 @@ function getRestaurants() {
     // Using Promise here to mimick API response from the backend
     return Promise.resolve(restaurants);
 }
+
+function getSignIn() {
```

Comment:
Rename function to clarify what the method returns. Also let's try to break this method down. It's too long and hence hard to read and understand

Reply:
ok